dll/directx/wine/d3dx9_36/d3dx9.cmake

```diff
@@ -0,0 +1,45 @@
+
+function(add_d3dx9_target __version)
+    set(module d3dx9_${__version})
+
+    spec2def(${module}.dll ${module}.spec ADD_IMPORTLIB)
+
+    list(APPEND SOURCE
```

Comment:
It's a bit sad that we now have to rebuild all of these files for each module.

I wonder if it would be worth it to create a library for files that don't depend on D3DX_SDK_VERSION and only build those once.

Reply:
I gave a thought about this, but finding which files should be separated is not trivial, as dxsdk is also concerned by this. And also, if wine introduces version related changes in those files, there is 99% chances that we wouldn't see it with this script.